sw/device/tests/pwrmgr_all_reset_reqs_test.c

```diff
@@ -0,0 +1,110 @@
+// Copyright lowRISC contributors.
```

Comment:
@jeoongp i'm a little bit confused by this test, how is this one different from `aon_timer_wdog_bite_reset_test.c`?

Reply:
This test does not include deep sleep waked up by watch dog. It tests the reset by watch dog timer.

Reply:
o got it. Should the test name be a bit different? This test name seems to imply all reset requests, but it looks like it's more focused on specifically on the wdog.